app/controllers/topics.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+  newTopic: Ember.computed(function () {
+    return this.store.createRecord('topic');
+  }),
+
+  replaceNewTopic () {
+    this.set('newTopic', this.store.createRecord('topic'));
```

Comment:
Yea I'm not really happy with this.  I don't know if it's a great idea to use the `newTopic` attribute to replace itself though.  This is the best I've got until a better thing comes along.